### ISSUE TITLE
OS X Build Fail (May be devkitARM issue)

### ISSUE THREAD

**Issue by TheDuckMan64** [delebile/arm9loaderhax#7]
When trying to compile, I always receive the issue:

make[1]: **\* No rule to make target `“/Users/edmonds_ha/devkitARM:??/ds_rules'.  Stop.
make: **\* [firm0] Error 2

This seems as though it might be a devkitARM issue though.

**Comment by pixel-stuck** [comment]
It's your setup, your path seems to be wrong (devkitARM:??) What are you using to set the path?

**Comment by jasondellaluce** [comment]
Can i close this?
Seems to not be a problem due to this software.

**Comment by mariogamer2** [comment]
I can say that it's work on linux.
As the file base_rule is probably the same for all devkit version,it's can be closed.

EDIT: @dark-samus yeah,it's path seem brocken.

The good path:

On MAC and Linux: /opt/devkitpro/
On Windows: C:/devkitpro/

**Comment by urherenow** [comment]
I've found (by building FBI) that there are differences in how paths are handled on Windows between make.exe versions. Most 3ds stuff works fine with gnu32 make v3.81. Before the official 2.0 rewrite, I had to use Gnu make v4.1 to get past path issues (but his version broke building everything else).

As of this moment, FBI builds fine using 3.81. 

TL;DR
Try a different make.exe from what you're using @TheDuckMan64

**Comment by TheDuckMan64** [comment]
Sorry I never answered, but the real issue was the way that OS X 10.11 (El Capitan) handles files, and with the added security and removal of ~/.bash_profile / ~/.bashrc, exporting something in my path was literally impossible. I sorted it out though :P
